docs/firmware/manual-flashing.md

```diff
@@ -6,19 +6,24 @@ Flashing your own firmware has the potential to brick your device. Do not do thi
 
 The steps below assume you are flashing an image named `coreboot.rom`; substitute it as necessary.
 
-1. Download flashrom, decompress, and ensure it is executable:
+1. Download flashrom and cbfstool, decompress, and ensure it is executable:
    * `wget -o flashrom.tar.gz https://mrchromebox.tech/files/util/flashrom_ups_libpci37_20240418.tar.gz`
    * `tar zxf flashrom.tar.gz`
    * `chmod +x flashrom`
+   * `wget https://mrchromebox.tech/files/util/cbfstool.tar.gz && tar -zxf cbfstool.tar.gz && chmod +x cbfstool`
+
 2. Flash your custom ROM
    * Backup your current firmware (just in case things go wrong):
      `sudo ./flashrom -p internal -r backup.rom`
-   * Flash your custom firmware: 
+   * Extract your VPD from your backup:
+         `./cbfstool backup.rom read -r RO_VPD -f vpd.bin`
+   * Inject the VPD into your custom ROM:
+         `./cbfstool coreboot.rom write -r RO_VPD -f vpd.bin`
+   * Extract your HWID and inject it into the custom ROM (if it exists)
+       * `./cbfstool backup.rom extract -n hwid -f hwid.txt` if this fails, don't run the following command
```

Comment:
this will only work if backup.rom is a MrChromebox image flashed by my script which contains hwid.txt